cmd/lncli/commands.go

```
@@ -393,6 +405,15 @@ func sendCoins(ctx *cli.Context) error {
 		displayAmt = balanceResponse.GetConfirmedBalance()
 	}
 
+	if ctx.IsSet("utxo") {
```

we need to make sure that there is no conflict with other flags here, like `sendall` and `amt`. We should fail early if there's a combination that doesn't make sense.

---

I think there is a check down there already.